### PR TITLE
Mark `entrypoints` as complete

### PR DESCRIPTION
**Pull request by sobolevn**
Source: https://github.com/takluyver/entrypoints

### CODE REVIEW

**Comment by sobolevn** [comment]
There's something wrong with GitHub lately. This my old commit message. Not the new one.

Probably because of the new code editor 🤔

**Comment by AlexWaygood** [comment]
I'm a big fan of this series of PRs, btw :)

**Comment by sobolevn** [comment]
Thanks!

Do you like the new UI? I am not quite sure yet.

**Comment by AlexWaygood** [comment]
> Do you like the new UI? I am not quite sure yet.

Not sure I've experienced it yet!

File: stubs/entrypoints/entrypoints.pyi [hunk 1]
```diff
@@ -23,6 +23,9 @@ class NoSuchEntryPoint(Exception):
     name: str
     def __init__(self, group: str, name: str) -> None: ...
 
+class CaseSensitiveConfigParser(ConfigParser):
+    optionxform = staticmethod(str)
```
Comment (AlexWaygood) [inline stubs/entrypoints/entrypoints.pyi:27]:
```suggestion
    optionxform: staticmethod[str]
```

**Review by AlexWaygood — APPROVED** [review]
Thanks!

**Pull request merged**